apps/zotonic_mod_wires/priv/lib/js/modules/livevalidation-1.3.js

```diff
@@ -282,6 +282,22 @@ LiveValidation.prototype = {
             return LiveValidation.TEXT;
         if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'HIDDEN')
             return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'COLOR')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'DATE')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'DATETIME-LOCAL')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'MONTH')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'WEEK')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'TIME')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'RANGE')
+            return LiveValidation.TEXT;
+        if (nodeName == 'INPUT' && this.element.type.toUpperCase() == 'SEARCH')
+            return LiveValidation.TEXT;
```

Comment:
I suppose that this can be simplify like:
1 - Define a constant eg.: 
```js
const x = ['COLOR', 'DATE', 'DATETIME-LOCAL', ''WEEK', ...']
```
2 - Use `includes()` method for detect if item are exist: eg:
```js
        if (nodeName == 'INPUT' && x.includes(this.element.type))
          ....
```

Reply:
I agree with you, @vkatsuba. This code can be improved, but IMHO, this lib needs to be updated, reviewed, or changed by another one, like [Zod](https://zod.dev/). The majority of the code is still the same as the [original one](https://github.com/alechill/livevalidation) (the last commit date is Feb 13, 2010). I just kept the code semantic.
See [this comment](https://github.com/zotonic/zotonic/issues/3305#issuecomment-1443368354) from @mworrell.

Reply:
Yep. It was just a proposal.